engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -213,36 +213,36 @@ public final HibernateValidatorConfiguration failFast(boolean failFast) {
 
 	@Override
 	public HibernateValidatorConfiguration allowOverridingMethodAlterParameterConstraint(boolean allow) {
-		this.methodValidationConfiguration.allowOverridingMethodAlterParameterConstraint( allow );
+		this.methodValidationConfigurationBuilder.allowOverridingMethodAlterParameterConstraint( allow );
 		return this;
 	}
 
 	public boolean isAllowOverridingMethodAlterParameterConstraint() {
-		return this.methodValidationConfiguration.isAllowOverridingMethodAlterParameterConstraint();
+		return this.methodValidationConfigurationBuilder.isAllowOverridingMethodAlterParameterConstraint();
```

Comment:
Not sure I'm really convinced by how we deal with the builder state in the getter here. The pattern is kinda weird. Not that I can think of something better though.